keps/sig-api-machinery/4008-crd-ratcheting/README.md

```diff
@@ -323,6 +333,113 @@ myOtherField: newly added field
 The feature proposed in this KEP would allow this change, since `myField` was
 unaltered from the original object. If `myField` was changed, the new value would have to pass the new validation rule to be written.
 
+### Nested Value Validations 
+
+OpenAPI-schemas support nested validations with logical operations:
+- not
+- oneOf
+- anyOf
+- allOf
+
+To keep the semantics simple and implementation clear, schemas and validations
+nested within the following will be not ratcheted:
+
+- not
+- oneOf
+
+Our intuition is that in the wild CRD authors do not make much use of them, and
+when they do the logic is complicated and references multiple fields. 
+
+For all these constructs, the conservative `DeepEqual` rule still applies 
+to the final output of the rule. 
+
+The following will be ratcheted with the conservative ratcheting rule, and
+their children will also be ratcheted.
+
+- allOf
+- anyOf
+
+### Atomic Lists and Maps
+
+Following Kubernets convention that fields annotated `x-kubernetes-list-type` or `x-kubernetes-map-type` as `atomic` change as a unit, the ratcheting rules
```

Comment:
```suggestion
Following Kubernetes convention that fields annotated `x-kubernetes-list-type` or `x-kubernetes-map-type` as `atomic` change as a unit, the ratcheting rules
```

Reply:
For atomic that's clear. Give the intuition why maps are handled like that too.

Reply:
Added a section about how altering `x-kubernetes-list-type` and `x-kubernetes-map-keys` does not get ratcheted

Reply:
Never mind. This comment was before ☕. Was reading map type for a list.